lib/SignIn.dart

```diff
@@ -0,0 +1,266 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:signup_module/Forgotpassword.dart';
+import 'package:signup_module/main.dart';
+
+class MySignInPage extends StatefulWidget {
+  const MySignInPage({Key? key}) : super(key: key);
+
+  @override
+  _MySignInPageState createState() => _MySignInPageState();
+}
+
+class _MySignInPageState extends State<MySignInPage> {
+  final GlobalKey<FormState> _formKey = GlobalKey<FormState>();
+
+  bool eye = true;
+
+  // final alphanumeric = RegExp('r^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]{8,}$');
```

Comment:
As mentioned above, Commented code should not be in the commit. It is a bad practice.